tests/Integrations/Mysqli/MysqliTest.php

```diff
@@ -23,13 +25,8 @@ public static function setUpBeforeClass()
     protected function setUp()
     {
         parent::setUp();
-        $this->setUpDatabase();
-    }
-
-    protected function tearDown()
-    {
         $this->clearDatabase();
-        parent::tearDown();
+        $this->setUpDatabase();
```

Comment:
The purpose of this change is to have data still available in the database after a test to help investigating why a test failed.